project_generator/generate.py

```diff
@@ -36,7 +36,12 @@ def generate(self, name = ''):
             if name != '':
```

Comment:
Might not be related to this change-set, but check please name argument and the variable name in the else clause. Then name parameter is not used at all, a bug? Line 48 in this file